tests/intg/test_slide_gen_chain.py

```
@@ -0,0 +1,258 @@
+
+        analysis_result = prompt_service.build_analysis_prompt(context)
+        assert isinstance(analysis_result, dict)
+        assert "prompt" in analysis_result
+        assert isinstance(analysis_result["prompt"], str)
+        assert len(analysis_result["prompt"]) > 0
+        assert "Test script content" in analysis_result["prompt"]
+
+    @patch(
+        "src.backend.chains.slide_gen_chain.print"
+    )  # Mock print to avoid output during tests
+    def test_error_handling_in_chain(self, mock_print, mock_template):
```

Ruff (0.12.2)

Unused method argument: `mock_print`

(ARG002)